spec/unit/mutant/context_spec.rb

```diff
@@ -31,27 +31,5 @@
 
       it { should eql(expected) }
     end
-
-    context 'with Class as scope' do
```

Comment:
Should these spec removals be broken up into a separate commit? I assume they were removed for some other reason?